### DISCUSSION TITLE
wire entangle + debounce

### DISCUSSION THREAD

**Discussion by usernotnull** [livewire/livewire#4526] [3. Feature ideas]
Currently, we can defer a livewire entangled property:
```
$wire.entangle('foo').defer
```
How about adding a .debounce also? My use case is in a multi-tag select component. I wanted to add a 500ms debounce if the user removes tags quickly. The only way around it is adding an update button and defer, but debouncing is much cleaner.

**Comment by joshhanley** [comment]
@usernotnull yeah that would be nice!

One way you can work around that is in Alpine, bind your multi-tag select to one Alpine property, and entangle a different property. 

Then you can watch the first property and when it changes, update the entangled property. That would allow you to put a debounce on that update process rather than having to add an `update` button.

Hope this helps!

**Comment by dayemsiddiqui** [comment]
@usernotnull I came across a similar problem, this might help you: https://github.com/livewire/livewire/discussions/7109#discussioncomment-9625309
